platform-api/src/main/java/io/datacater/core/config/ConfigEndpoint.java

```diff
@@ -0,0 +1,82 @@
+package io.datacater.core.config;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import io.datacater.core.exceptions.ConfigNotFoundException;
+import io.smallrye.mutiny.Uni;
+import java.util.List;
+import java.util.UUID;
+import javax.inject.Inject;
+import javax.ws.rs.*;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import org.eclipse.microprofile.openapi.annotations.parameters.RequestBody;
+import org.hibernate.reactive.mutiny.Mutiny.SessionFactory;
+
+@Path("/config")
```

Comment:
@flippingbits @HknLof should we always use the plural here?
If so, pipelines actually also only uses the singular form, which we should probably change too

Reply:
Yes, if we are dealing with resources, like in this case, we should use the plural.